plugins/lsp/language-servers/default.nix

```
@@ -621,8 +621,9 @@ let
       settingsOptions = import ./tinymist-settings.nix { inherit lib helpers; };
     }
     {
-      name = "tsserver";
-      description = "tsserver for TypeScript";
+      name = "ts-ls";
+      serverName = "ts_ls";
+      description = "ts-ls for TypeScript";
```

```suggestion
      description = "ts_ls for TypeScript";
```

The actual server is ts_ls, even though our module uses snake case.